src/noframe.js

```diff
@@ -0,0 +1,40 @@
+export default function noframe(target, container) {
+  let els = typeof target === 'string' ? document.querySelectorAll(target) : target;
+  if (!('length' in els)) {
+    els = [els];
+  }
+  for (let i = 0; i < els.length; i++) {
```

Comment:
I'll check again but I didn't go that direction b/c of the way babel compiles `els.forEach`.